deploy/azuredeploy.json

```diff
@@ -61,7 +61,12 @@
     "postCallSurveyOneQuestionPollPrompt": "",
     "postCallSurveyOneQuestionPollType": "",
     "postCallSurveyOneQuestionPollAnswerPlaceholder": "",
-    "postCallSurveyOneQuestionPollSaveButtonText": ""
+    "postCallSurveyOneQuestionPollSaveButtonText": "",
+    "cosmosDbAccountName": "[concat('db-account-', parameters('appServiceName'))]",
+    "roleDefinitionName": "[concat('sql-role-definition-', parameters('appServiceName'))]",
+    "cosmosDbName": "",
```

Comment:
I think we should provide a default value here (other variables also have a default value). Since this is not a parameter, it is easily missed and while the resources will be successfully created, the db name will be left empty in the App Service's environment variables.

Reply:
Agreed, added a default value.